packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/sharedobjects/SharedRef.kt

```diff
@@ -0,0 +1,7 @@
+package expo.modules.kotlin.sharedobjects
+
+/**
+ * Shared object (ref) that holds a strong reference to any native object. Allows passing references
+ * to native instances among different independent libraries.
+ */
+open class SharedRef<RefType>(val ref: RefType) : SharedObject()
```

Comment:
On iOS it's a `pointer`, not `ref`. Imho we should decide on how to name it and keep it the same on both platforms.

Reply:
I was aware that you named that pointer. However, I wouldn't say I like that name in the Android context. The `ref` sounds more user-friendly to me 😅